spectator-api/src/main/java/com/netflix/spectator/api/AbstractRegistry.java

```diff
@@ -51,9 +52,23 @@ public AbstractRegistry(Clock clock) {
    *     Configuration settings for the registry.
    */
   public AbstractRegistry(Clock clock, RegistryConfig config) {
+    this(clock, config, new ConcurrentHashMap<>(1024));
```

Comment:
This is setting the initial size not the concurrency level. Can you create a [jmh test](https://github.com/Netflix/spectator/tree/master/spectator-api/src/jmh/java/com/netflix/spectator/perf) showing the impact?